test/unit/spec/narrative_core/jobCommChannel-spec.js

```diff
@@ -130,21 +135,49 @@ define(['jobCommChannel', 'base/js/namespace', 'common/runtime', 'testUtil'], (
             {
                 channel: 'request-job-log',
                 message: { jobId: 'someJob', options: {} },
-                type: 'job_logs',
+                expected: {
+                    target_name: 'KBaseJobs',
+                    request_type: 'job_logs',
+                    job_id: 'someJob',
+                },
             },
             {
-                channel: 'request-latest-job-log',
+                channel: 'request-job-log-latest',
                 message: { jobId: 'someJob', options: {} },
                 type: 'job_logs_latest',
             },
+            {
+                channel: 'request-job-log-latest',
```

Comment:
add in some tests to check the new merging of `message.options` object is working OK